benchopt/utils/github.py

```diff
@@ -26,7 +26,11 @@ def publish_result_file(benchmark_name, file_path, token):
             f"Could not upload file {file_to_upload}."
         )
 
-    file_content = pd.read_parquet(file_to_upload).to_csv()
+    if file_to_upload.suffix == ".csv":
+        with file_to_upload.open('r') as f:
+            file_content = f.read()
+    else:
+        file_content = pd.read_parquet(file_to_upload).to_csv()
```

Comment:
Isn't is possible to keep parquet file here? you shoudl probably read it in binary but it would be more convenient to upload the results in a compress format.

Moreover, this would produce deprecation warning (we use the same code to generate result on the website and locally).

Reply:
agreed. We should upload the parquet files